back/src/main/java/net/daw/alist/models/User.java

```diff
@@ -113,7 +113,7 @@ public void setImage(String imagePath) throws IOException, SQLException {
       imagePath = "static/images/notFound.jpg";
     }
     this.image = pathToImage(imagePath);
-    this.imagePath = imagePath.replace("static", "");
+    this.imagePath = imagePath.replace("static","");
```

Comment:
why not space? :(

Reply:
Must've been prettier. Idk, or I accidentally did it xd